netbeans.apache.org/src/content/templates/blogindex.gsp

```diff
@@ -0,0 +1,64 @@
+<%/*
+
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+*/%>
+<%/*
+
+    page_noaside.gsp: A main area with no aside.
+
+*/%>
+<!doctype html>
+<html class="no-js" lang="en" dir="ltr">
+  <%include "templatesparts/head.gsp"%>
+  <body>
+    <%include "templatesparts/menu.gsp"%>
+    <%include "templatesparts/news.gsp"%>
+    <div class='grid-container main-content'>
+      <h1 class="sect1">${content.title}</h1>
+      <%
+// content by aggregation 
+    TreeMap sectionsubitm = [:]
+    for ( atag in tags ) {
+        if ( atag.name=="blogentry") {
+            for ( mydoc in atag.tagged_documents ) {
+                sectionsubitm.put(mydoc.date,mydoc);
+            }
+        }
+    }
+    sectionsubitm.sort();
+    out.println('<div class="ulist">');
+    out.println('<ul>');
+    for (adoc in sectionsubitm) {
+        out.println('<li><p>');
+        out.println('<a href="'+content.rootpath+adoc.value.uri+'">'+adoc.value.title+' '+adoc.value.date+'</a>');
+        out.println('</p></li>');
+    }
+    out.println('</ul>');
+    out.println('</div>');
+    out.println('</div>');
```

Comment:
I see two closing `</div>` s here, but I can only see one opened  `<div>` above. If you're trying to close the `<div>` on line 32 above, why not use a plain `</div>` there? Isn't that more legible?

Reply:
Ah, I see `feed.ftl` is required by jBake. Cool.

Reply:
thanks div is not needed, come from a copy paste. I use the ftl over gsp to get it fast because I've seen much more example of ftl. Hope mixing template is not much an issue